loopchain/blockchain/blockchain.py

```diff
@@ -317,7 +316,51 @@ def __add_block(self, block: Block, confirm_info, need_to_write_tx_info=True, ne
 
             return True
 
-    def __write_block_data(self, block: Block, confirm_info):
+    def __add_tx_to_block_db(self, block, receipts, batch=None):
+        """block db 에 block_hash - block_object 를 저장할때, tx_hash - block_hash 를 저장한다.
```

Comment:
Writing it in english is needed.

Reply:
I didn't write it in this PR. but I will change it.